### DISCUSSION TITLE
스터디 운영에 관한 회고록

### DISCUSSION THREAD

**Discussion by fkdl0048** [fkdl0048/Mentoring-Project#105] [General]
멘토링에 진행한 스터디 관련 내용에 대한 회고록을 작성했습니다..

- [링크](https://fkdl0048.github.io/daily/StudyRetrospect/)

**Comment by jongfeel** [comment]
담백하게 잘 작성하셨네요.

앞으로 어떻게 하면 좋을지에 대한 계획을 천천히 세워 보신 후에
다시 실천해서 경험해 보신 후 그 경험에 대한 회고를 반복해서 써 보시면 어떨까 합니다.

**Reply by fkdl0048** [reply]
넵 그래서 이번 회차는 인터뷰도 진행하고 너무 딱딱한 내용으로 시작하는게 아닌 개발자 마인드셋에 관련된 책으로 시작하려고 합니다.

제가 시작했던 것처럼 책을 읽으면 좋은 이유와 좋은 개발자가 되기 위한 생각을 같이 공유하는 게 좋을 것 같다고 생각한 것 같습니다..

*멘토님이 추천해주신 프로그래머의 길, 멘토에게 묻다와 같이 이번 회차는 책을 좋아하시는 분들과 클린코더를 읽어보려고 합니다!*

짧은 책으로 좀 성취감과 퍽퍽하지 않은 내용으로 시작하면 좋을 것 같다는 생각으로 선정했습니다..!

나중엔 아카데미 컨퍼런스처럼 꾸준하게 인사이트를 나눌 동료를 만드는 것이 목표입니다!

이후에 다시 한번 회고록을 작성해보면 달라진게 멍확하겠죠..!
